tox.ini

```diff
@@ -1,21 +1,28 @@
+[flake8]
+exclude =
+    .git
+    __pycache__
+    docs/source/conf.py
+    old
+    build
+    dist
+    *site-packages/
+    *bin/
+
 [tox]
-envlist = {py27,py36}-sklearn{17,18,19}-pandas{19,22}
+envlist = {py36}-sklearn{23}-pandas{1}
```

Comment:
Don't we want to test with python 3.7 and 3.8, and perhaps some other still supported version of sklearn and pandas?

Reply:
yes..I will add them as well.